server/public/model/config.go

```diff
@@ -434,6 +434,7 @@ type ServiceSettings struct {
 	MaximumPayloadSizeBytes                           *int64  `access:"environment_file_storage,write_restrictable,cloud_restrictable"`
 	MaximumURLLength                                  *int    `access:"environment_file_storage,write_restrictable,cloud_restrictable"`
 	ScheduledPosts                                    *bool   `access:"site_posts"`
+	EnableLateTimeWarnings                            *bool   `access:"site_users_and_teams"`
```

Comment:
The timeframe where the warning is shown is 10 PM until 6 AM. Should we rename the setting to `EnableNightTimeWarnings` or `EnableNonWorkHoursWarnings` to make it clear that it cover not only late hours but also the night?

Reply:
I don't think we are identifying here "non working hours", we are identifying late hours. I don't know how is the vision in the future, but this is not about "this person is not working", this is about "Is pretty late for this person", what is different. @matthewbirtch what do you think about this? Any preferences about the naming, specially with the future of the feature in mind?

Reply:
For the future, I think we may actually add a working hours user setting that would allow users to set their working hours. We would leverage the working hours to determine when to show this notice if it's outside those hours. So I actually think `EnableNonWorkHoursWarnings` makes sense.